widgetssdk/src/main/java/com/glia/widgets/messagecenter/MessageCenterController.kt

```diff
@@ -50,6 +53,19 @@ internal class MessageCenterController(
 
         view?.emitUploadAttachments(getFileAttachmentsUseCase.execute())
         view?.setupViewAppearance()
+
+        updateAllowFileSendState()
+    }
+
+    private fun updateAllowFileSendState() {
+        siteInfoUseCase.execute { siteInfo: SiteInfo?, _ ->
+            onSiteInfoReceived(siteInfo)
+        }
+    }
+
+    private fun onSiteInfoReceived(siteInfo: SiteInfo?) {
+        val attachmentAllowed = siteInfo == null || siteInfo.allowedFileSenders.isVisitorAllowed
```

Comment:
NIT:
```suggestion
        val attachmentAllowed = siteInfo?.allowedFileSenders?.isVisitorAllowed ?: true
```
But maybe it would be better to hide attachment button when `siteInfo` is `null`

Reply:
`val attachmentAllowed = siteInfo?.allowedFileSenders?.isVisitorAllowed ?: false`
So, if `siteInfo` is `null` then `attachmentAllowed` should be `false`, instead of `true`, isn't it?